benchmarks/cloudsuite3/graph-analytics/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Setup
 
-Use the tools/docker_prep [README](/tools/docker_prep/README) to stage the docekr iamge used in this benchmark test.
+Use the tools/docker_prep [README](/tools/docker_prep/README) to stage the docekr image used in this benchmark test.
```

Comment:
Docker is misspelled. Replace `docekr` with `docker` in all files with the same problem.